cilium-dbg/cmd/ip_get.go

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	ipApi "github.com/cilium/cilium/api/v1/client/policy"
```

Comment:
nit: can you remove the capital in the alias for `ipapi` (cf https://go.dev/blog/package-names)